Cargo.toml

```diff
@@ -68,9 +91,8 @@ serde_cbor = { version = "0.11", optional = true }
 bincode = { version = "1.3", optional = true }
 frunk = { version = "0.3", optional = true }
 
-teloxide-macros = { git = "https://github.com/teloxide/teloxide-macros", branch = "master", optional = true }
-
 [dev-dependencies]
+teloxide-core = { version = "0.2.1", features = ["auto_send"] }
```

Comment:
Without `resolver = "2"` I assume this will activate the feature even when not built in dev mode. (it's a known cargo bug or something)
```suggestion
```